docs/File-Formats.md

```diff
@@ -126,10 +126,10 @@ The first four rows of the clinical data file contain tab-delimited metadata abo
 - Row 3: **The attribute Datatype**: The datatype of each clinical attribute (must be one of:  STRING, NUMBER, BOOLEAN)
 - Row 4: **The attribute Priority**: A number which indicates the importance of each attribute.  In the future, higher priority attributes will appear in more prominent places than lower priority ones on relevant pages (such as the [Study View](http://www.cbioportal.org/study?id=brca_tcga)). A lower number indicates a higher priority.
     ```
-    To promote certain chart in study view, please increase priority in the database to certain number. The higher the score, the higher priority it will be displayed in the study view.
+    To promote certain chart in study view, please increase priority to certain number. The higher the score, the higher priority it will be displayed in the study view.
```

Comment:
I just realized that this sentence contradicts the previous sentence above: "A lower number indicates a higher priority.". Please update the previous sentence as well to make it consistent.